website/client/components/userMenu/profile.vue

```diff
@@ -636,6 +630,17 @@ export default {
       selectedPage: 'profile',
       achievements: {},
       content: Content,
+      equipTypes: {
```

Comment:
where are these used?

Reply:
https://github.com/HabitRPG/habitica/pull/9887/files#diff-296cf9ad5234aa0fac0c65fafc0c6b97R126

to list each type of gear, so that we don't have to write the same popover logic per item type ^^